.golangci.yml

```diff
@@ -1,11 +1,13 @@
 version: "2"
 linters:
+  enable:
+    - errorlint
```

Comment:
Compare https://github.com/containers/image/pull/2512 . As in there, adding this linter only adds unnecessary work. (`%#v` vs. `%#w` makes no difference at the CLI level.)

Reply:
Removed the second patch. I'm slightly concerned about not being able to lint for missing `%w` in other parts of the code, but perhaps this is not a big deal.